packages/trie/src/types.ts

```diff
@@ -60,13 +54,20 @@ export interface TrieOpts {
    * Store the root inside the database after every `write` operation
    */
   useRootPersistence?: boolean
+
+  /**
+   * Flag to prune the trie. When this is set to `true`,
+   * each time a value is overridden, then the (intermediate) nodes which are now not
+   * reachable anymore will be pruned (deleted) from the trie
```

Comment:
```suggestion
   * Flag to prune the trie. When set to `true`, each time a value is overridden,
   * unreachable nodes will be pruned (deleted) from the trie
```
A bit more concise imo